<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>predication</class>
 <widget class="QDialog" name="predication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>选择模型：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="modelEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnModel">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>预测数据：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dataEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnData">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,2,1,2,1,1,2,2,1,2,1">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>属性：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="attributel"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>列—</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="attributer"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>列</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>标签数据：</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="tagl"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>列—</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="tagr"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>列</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1,1,1,4,2,4,2">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>训练集：验证集：测试集 = </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="train"/>
     </item>
     <item>
      <widget class="QSpinBox" name="check"/>
     </item>
     <item>
      <widget class="QSpinBox" name="test"/>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>epochs = </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="epochs"/>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>batch_size = </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="batch_size"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPlainTextEdit" name="log"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btncancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
